toolkit/scripts/toolchain/container/toolchain_build_temp_tools.sh

```diff
@@ -125,9 +125,9 @@ rm -rf CBL-Mariner-Linux-Kernel-rolling-lts-mariner-5.15.32.1
 
 touch $LFS/logs/temptoolchain/status_kernel_headers_complete
 
-echo glibc-2.34
-tar xf glibc-2.34.tar.xz
-pushd glibc-2.34
+echo glibc-2.35
+tar xf glibc-2.35.tar.xz
+pushd glibc-2.35
 mkdir -v build
```

Comment:
Question for this one: should we also apply the `glibc-2.35-fhs-1.patch` like what we do inside `toolchain_build_in_chroot.sh` or we don't need to worry about that here, since this is just the 1<sup>st</sup> stage of the toolchain?

Reply:
I'm not sure it's required, but it should not hurt to patch here as well. I'll update this

Reply:
BTW, LFS previously did not apply this patch during the first build of glibc (see: https://www.linuxfromscratch.org/lfs/downloads/9.1-systemd/LFS-BOOK-9.1-systemd-NOCHUNKS.html#ch-tools-glibc). It looks like they do apply the patch in the latest version (see: https://www.linuxfromscratch.org/lfs/downloads/11.0-systemd/LFS-BOOK-11.0-NOCHUNKS.html#ch-tools-glibc)